packages/destination-actions/src/destinations/launchdarkly-audiences/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const CONSTANTS = {
+  // TODO: update to prod URLs when testing is complete
```

Comment:
kept this as staging URLs for PR review to test on staging, but will update before merging and try to make sure we can update the flags on prod

Reply:
updated to prod ✅